packages/okta-adapter/test/filters/app_logo.test.ts

```diff
@@ -101,6 +103,17 @@ describe('app logo filter', () => {
       expect(res.errors).toHaveLength(1)
       expect(res.errors?.[0].message).toEqual('Failed to fetch App logo. Failed to find content type for app1')
     })
+    // Reproduce an issue where the filter does not give unique file names to AppLogo instances if their names only
+    // differed by underscores.
+    it('should give AppLogo instances unique file names', async () => {
+      // Same name, but with underscore.
+      const anotherAppInstance = createAppInstance('app_1')
+      const elements = [appType, appInstance, anotherAppInstance].map(e => e.clone())
```

Comment:
nit: Can remove the clone() for each element 
```suggestion:
const elements = [appType, appInstance, anotherAppInstance]
```